tests/js/environment.js

```diff
@@ -1,23 +1,28 @@
 var fluid = require("infusion");
 
-fluid.defaults("gpii.pouch.tests.environment", {
+fluid.defaults("gpii.test.pouch.environment", {
     gradeNames: ["fluid.test.testEnvironment"],
     port:       6792,
     baseUrl:    "http://localhost:6792/",
     testUrl:    "/sample/",
     events: {
-        constructServer: null,
-        onStarted: null
+        constructFixtures: null,
+        onHarnessReady: null,
+        onFixturesConstructed: {
+            events: {
+                onHarnessReady: "onHarnessReady"
+            }
+        }
```

Comment:
An options distribution would be useful here to distribute the user's exact grade to the harness component - otherwise they will just write one themselves